app/src/test/java/org/oppia/android/app/testing/options/ReadingTextSizeFragmentTest.kt

```diff
@@ -151,46 +167,57 @@ class ReadingTextSizeFragmentTest {
     )
   }
 
-  private fun clickSeekBar(position: Int): ViewAction {
-    return GeneralClickAction(
-      Tap.SINGLE,
-      CoordinatesProvider { view ->
-        val seekBar = view as SeekBar
-        val screenPos = IntArray(2)
-        seekBar.getLocationInWindow(screenPos)
-        val trueWith = seekBar.width - seekBar.paddingLeft - seekBar.paddingRight
-
-        val percentagePos = (position.toFloat() / seekBar.max)
-        val screenX = trueWith * percentagePos + screenPos[0] + seekBar.paddingLeft
-        val screenY = seekBar.height / 2f + screenPos[1]
-        val coordinates = FloatArray(2)
-        coordinates[0] = screenX
-        coordinates[1] = screenY
-        coordinates
-      },
-      Press.FINGER, /* inputDevice= */ 0, /* deviceState= */ 0
-    )
-  }
-
-  private fun seekBarProgress(progress: Int): TypeSafeMatcher<View> {
+  private fun textViewSize(expectedSize: Float): TypeSafeMatcher<View> {
     return object : TypeSafeMatcher<View>() {
       override fun describeTo(description: Description?) {
-        description?.appendText("SeekBarProgress")
+        description?.appendText("TextView with fontSize: $expectedSize")
       }
 
       override fun matchesSafely(item: View?): Boolean {
-        return (item as SeekBar).progress == progress
+        val textView = item as TextView
+        val pixels = textView.textSize
+        val actualSize = pixels / textView.resources.displayMetrics.scaledDensity
+        return expectedSize.compareTo(actualSize) == 0
       }
     }
   }
 
-  private fun checkTextSize(value: Int) {
-    onView(withId(R.id.reading_text_size_seekBar)).check(matches(seekBarProgress(value)))
+  private fun checkTextSize(index: Int) {
```

Comment:
this function we are only checking if the item is checked or not, we are not checking the text size. 
Please update the function name.

Reply:
done